libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -2520,16 +2520,24 @@ void IRGeneratorForStatements::appendExternalFunctionCall(
 		</noTryCall>
 		<?+retVars> let <retVars> </+retVars>
 		if <success> {
+			let <returnDataSize> := <estimatedReturnSize>
```

Comment:
Maybe using
```suggestion
			let <returnDataSize>
```
here and then putting the assignment into the `!dynamicReturnSize` part is easier to understand?